src/_data/schedule/day_two.yml

```diff
@@ -0,0 +1,54 @@
+- start_at: "8:00"
+  end_at: "9:00"
+  title: "Conference Door Opens"
+- start_at: "9:00"
+  end_at: "9:10"
+  title: "Welcome Address"
+- start_at: "9:10"
+  end_at: "9:55"
+  title: "Ruby & JVM: A (JRuby) Love Story"
```

Comment:
i think the keynotes should be prefixed "Keynote: " or somehow otherwise highlighted